Cargo.toml

```diff
@@ -10,13 +10,20 @@ members = [
 version = "0.15.0"
 
 [workspace.dependencies]
+anyhow = "1.0"
+aurora-engine = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0" }
+aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.7.0" }
 # TODO: Publish and use version from github / crates.io
 aurora-sdk = { git = "https://github.com/aurora-is-near/aurora-contracts-sdk.git", rev = "31e874d69d8e53dd4931aa1fbabe42e4bcef5c03" }
 base64 = "0.13.0"
+hex = "0.4"
 near-contract-standards = "4.0.0"
 near-sdk = "4.0.0"
 serde = "1"
+tokio = { version = "1.18", features = ["full"] }
+toml = "0.5"
 uint = { version = "0.9.3", default-features = false }
+workspaces = { git = "https://github.com/near/workspaces-rs.git", rev = "8b3356a29f71756d86840679d3ff162b885123c9" }
```

Comment:
Why do we need this particular version?

Reply:
The currently published version, 0.5.0, [depends on a pre-release version (0.4.0-beta.0)](https://github.com/near/workspaces-rs/blob/0.5.0/workspaces/Cargo.toml#L37) of the near JSON RPC client, however the official 0.4.0 version of that library has now [been released](https://crates.io/crates/near-jsonrpc-client/versions), and the two versions are not compatible (i.e. there was a breaking change in the library between 0.4.0-beta.0 and 0.4.0). Unfortunately, cargo thinks that these versions are interchangable and will automatically choose version 0.4.0 as the dependency of `near-jsonrpc-client` when you use version 0.5.0 of `workspaces`, and the result is a compilation error.

The [commit I am referencing](https://github.com/near/workspaces-rs/commit/8b3356a29f71756d86840679d3ff162b885123c9) in our project here includes a fix where I have updated workspaces to use the 0.4.0 release of the JSON RPC. This fixes the compilation error we get with version 0.5.0. We'll have to change this git dependency to version 0.6.0 when they eventually release it.